src/reuse/supported_licenses.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2021 Free Software Foundation Europe e.V. <https://fsfe.org>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""supported-licenses command handler"""
+
+import sys
+
+from ._licenses import _LICENSES, _load_license_list
+from .project import Project
+
+# pylint: disable=unused-argument
+
+
+def add_arguments(parser) -> None:
+    """Add arguments to the parser."""
+
+
+def run(args, project: Project, out=sys.stdout):
+    """Print the supported SPDX licenses list"""
+
+    licenses = _load_license_list(_LICENSES)[1]
+
+    for license_id, license_info in licenses.items():
```

Comment:
Since this is trivial I didn't create a dedicated module returning the licence lines.